gateway/starknet.go

```diff
@@ -44,9 +44,6 @@ func (sg *Gateway) Call(ctx context.Context, call types.FunctionCall, blockHashO
 	if len(call.Calldata) == 0 {
 		call.Calldata = []string{}
 	}
-	if len(call.Signature) == 0 {
-		call.Signature = []string{"0", "0"} // allows rpc and http clients to implement(has to be a better way)
```

Comment:
It looks like this is the culprit of the error: 
`500: StarkErrorCode.MALFORMED_REQUEST __init__() missing 1 required positional argument: 'signature'`

So the `account.AccountNonce` call fails in `EstimateFee`